schema/defs.json

```diff
@@ -18,8 +18,8 @@
         },
         "int64": {
             "type": "integer",
-            "minimum": -9223372036854775808,
-            "maximum": 9223372036854775807
+            "minimum": -9223372036854776000,
+            "maximum": 9223372036854776000
```

Comment:
?

Reply:
It's weird. I found that we need above +1.7. I'll fix it. Thanks.
https://github.com/jqlang/jq/issues/1741#issuecomment-1712181286